.github/workflows/gradle.yml

```diff
@@ -105,3 +105,41 @@ jobs:
           java-version: 21
       - name: spotless
         run: ./gradlew --no-daemon --parallel clean spotlessCheck
+
+  go-corset-tests:
+    runs-on: ubuntu-latest
```

Comment:
maybe should use `ubuntu-latest-128` ? See with @FlorianHuc

Reply:
So, the currently `gradle.yml` file only uses `ubuntu-latest` (see [here](https://github.com/Consensys/linea-tracer/blob/arith-dev/.github/workflows/gradle.yml)).  I don't think Tsvetan has got it setup yet so that we can use e.g. `ubuntu-latest-16core`.

Reply:
Ok, I see that on the long running branch it has been changed to `ubuntu-latest-128`.

Reply:
yes, by @FlorianHuc  : https://github.com/Consensys/linea-tracer/pull/748/files#diff-0d634959c8276ae976e39ba475e63d0b9ec27824470c79f1971f58fe37c46325

Reply:
Ok, yeah will check with Florian now.

Reply:
So, this has resolved itself with the merge from the long running branch.  I am now using `ubunutu-latest-128`.